lib/gooddata/lcm/actions/provision_clients.rb

```diff
@@ -38,11 +38,13 @@ def call(params)
           domain_name = params.organization || params.domain
           domain = client.domain(domain_name) || fail("Invalid domain name specified - #{domain_name}")
 
+          data_product = params.data_product
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)